pkg/webhook/mutation/pod_mutator/oneagent_mutation/config.go

```diff
@@ -22,8 +22,9 @@ const (
 	oneAgentShareVolumeName   = "oneagent-share"
 	injectionConfigVolumeName = "injection-config"
 
-	oneAgentCustomKeysPath = "/var/lib/dynatrace/oneagent/agent/customkeys"
-	customCertFileName     = "custom.pem"
+	oneAgentCustomKeysPath  = "/var/lib/dynatrace/oneagent/agent/customkeys"
+	customCertFileName      = "custom.pem"
+	customProxyCertFileName = "custom_proxy.pem"
```

Comment:
can't you use the const you created in [pkg/consts/agent_injection.go](https://github.com/Dynatrace/dynatrace-operator/pull/2938/files#diff-eda55294e0cf89ab2b60121173dcbb8384342001ed988214c5b3d4590f73012eR35-R36) ?